artview/components/select_region.py

```diff
@@ -342,7 +342,7 @@ def closeEvent(self, QCloseEvent):
         self.disconnect()
         super(SelectRegion, self).closeEvent(QCloseEvent)
 
-    def newPlotAxes(self, variable, value, strong):
+    def NewPlotAxes(self, variable, strong):
         self.disconnect()
         if self.VplotAxes.value is not None:
             self.fig = self.VplotAxes.value.get_figure()
```

Comment:
Select Region is not working in my tests. When selecting *Tools -> Select Region of Interest* from a Display window I get the following error:
```TypeError: _get_x_y_z() takes exactly 4 arguments (5 given)```

When using GateFilter, no operations are performed